src/mcmc/emcee.jl

```diff
@@ -133,10 +134,9 @@ function AbstractMCMC.bundle_samples(
     le = getlogevidence(samples, state, spl)
 
     # Set up the info tuple.
+    info = (varname_to_symbol = OrderedDict(zip(varnames, varnames_symbol)),)
```

Comment:
Should we make this inclusion optional? Forcing it could be annoying for serialization of the chains, etc. @yebai

Reply:
If we skip it in serialization, then those methods won't work after deserialization anyways :confused: 

And yes I agree it should be enabled by default, but I was also thinking it would be useful to provide a way to disable it.

Reply:
You can, but it means that you no longer can do `using MCMCChains; deserialize(...)` but you need to have done `using Turing` first. IMO this is quite annoying :confused:

Reply:
Potentially in the longer run, but I don't think we should do that when MCMCChains doesn't use any functionality from AbstractPPL.